docs/swagger.json

```
@@ -20,7 +20,69 @@
                 "summary": "Returns details of the configuration of eBPF Programs for all interfaces on a node",
                 "responses": {
                     "200": {
-                        "description": ""
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/l3af/configs/v1/add": {
+            "post": {
+                "description": "Adding new eBPF Programs on node",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary": "Adding new eBPF Programs on node",
+                "parameters": [
+                    {
+                        "description": "BPF programs",
+                        "name": "cfgs",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/models.L3afBPFPrograms"
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/l3af/configs/v1/delete": {
+            "post": {
+                "description": "remove eBPF Programs on node",
```

```suggestion
                "description": "Removes eBPF Programs on node",
```
Use grammar and capitalization consistent with line 13